changelogs/CHANGELOG-1.7.md

```diff
@@ -74,6 +74,19 @@ This command collects information about a Velero installation, such as pod logs
     from when it is in the UploadingPartialFailure state.
   * 🐛 Fix plugin name derivation from image name (#3711, @ashish-amarnath)
   * ✨ ⚠️ Remove CSI volumesnapshot artifact deletion
+  *  - fix buggy pager func
```

Comment:
Let's create a separated changelog file for this PR just like what we do for the main branch. We'll collect all of the unreleased changelogs and merge them into here when releasing

Reply:
wonder if this should be updated https://velero.io/docs/v1.5/code-standards/#adding-a-changelog to reflect that for release branches the changelog process is the same as main

Reply:
Opened a new issue #4333 to track this